Added READMe.md files and some minor changes (#5816)
umbraco/UmbracoDocs: sustainability-best-practices/frontend/README.md

---
description: >-
  Here you can learn about how to optimize assets, implement a mobile-first strategy and other suggestions.
---

# Overview

In this section, you can find some tips to optimize your website's performance and reduce its carbon footprint:

## Page Weight Budget

Choose how to transfer data for web page access minimizing the impact on carbon emissions.

## Image Optimization

> Learn how to optimize images before uploading to a web page and usage of different file formats.

Reviewer: it is not really a step by step guide but more like suggestions on what tools one can use etc
```suggestion
Optimize images before uploading to a web page and make use of different file formats.
```

## Asset Optimization

> Learn how to improve website performance, reduce carbon impact, and enhance user experience.

Reviewer: My suggestion of an overview of this article:
```suggestion
Improve website performance by optimizing fonts, minifying assets and cache static assets.
```

## Limit use of 3rd parties

To avoid excessive HTTP requests and potential data sharing, limit the use of third-party resources on websites.

## Lazyload Resources

> Learn how to optimize the loading of web pages using Lazyload resources.

Reviewer: ```suggestion
Optimize the loading of web pages by using Lazyload resources.
```
